include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.cpp

```diff
@@ -171,10 +173,24 @@ TimeIntBDF<dim, Number>::ale_update()
 
 template<int dim, typename Number>
 void
-TimeIntBDF<dim, Number>::advance_one_timestep_partitioned_solve(bool const use_extrapolation)
+TimeIntBDF<dim, Number>::advance_one_timestep_partitioned_solve(bool const use_extrapolation,
+                                                                bool const update_velocity,
+                                                                bool const update_pressure)
 {
   this->use_extrapolation = use_extrapolation;
   this->store_solution    = true;
+  this->update_velocity   = update_velocity;
+  this->update_pressure   = update_pressure;
+
```

Comment:
checking if one uses `BDFCoupledSolution` and a semi-implicit method could be done at setup, but only with the combined `Parameters` from fluid and FSI. This is did not do _yet_, but put the assert here.